index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```

Comment:
The file structure is not quite what we would expect from a vue application. I am assuming you have started this as a vanilla app, then started converting it to vue. If you need help with structuring an application using vue please reach out to one of the helpers :)